CHANGELOG.md

```diff
@@ -35,7 +35,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 # Changelog
 
 ## [Unreleased]
-* 
+
+## [v9.0.2-rc0] - 2023-03-26
+* (feat) Bump [Interchain-Security](https://github.com/cosmos/interchain-security) [v1.1.0](https://github.com/cosmos/interchain-security/releases/tag/v1.1.0) provider module. See the [release notes](https://github.com/cosmos/interchain-security/releases/tag/v1.1.0) for details.
+* (gaia) Bump [cosmos-sdk](https://github.com/cosmos/cosmos-sdk) to [v0.45.15-ics](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.45.15-ics). See [CHANGELOG.md](https://github.com/cosmos/cosmos-sdk/blob/release/v0.45.x/CHANGELOG.md) for details.
+* (gaia) Replace [tendermint](https://github.com/informalsystems/tendermint) to [comet](https://docs.cometbft.com/v0.34/). See [CHANGELOG.md](https://github.com/cometbft/cometbft/blob/v0.34.27/CHANGELOG.md#v03427) for details.
```

Comment:
Let's keep v9.0.2 only with the bump of ICS. We cut afterwards v9.0.3 with the replace to Comet.